CountryPickerView/CountryPickerViewController.swift

```diff
@@ -173,6 +173,14 @@ extension CountryPickerViewController {
     override public func tableView(_ tableView: UITableView, sectionForSectionIndexTitle title: String, at index: Int) -> Int {
         return sectionsTitles.firstIndex(of: title)!
     }
+
+    public override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return CGFloat.leastNormalMagnitude
+    }
+
+    public override func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+       return CGFloat.leastNormalMagnitude
```

Comment:
Wrong indentation. You can also drop `return CGFLoat.` like so
```suggestion
        .leastNormalMagnitude
```